src/api/app/controllers/webui/project_controller.rb

```diff
@@ -655,6 +655,8 @@ def load_project_info
 
     reqs = @project.open_requests
     @requests = (reqs[:reviews] + reqs[:targets] + reqs[:incidents] + reqs[:maintenance_release]).sort!.uniq
```

Comment:
if we still need  the variable ```@requests``` wouldn't be better to have it as ```@requests = @incoming_requests + @outgoing_requests``` to avoid to recalculate it again?

Reply:
Yes, I think we need `|` though as they could be duplicated.